.github/workflows/weekly_mac_ci.yml

```diff
@@ -56,4 +56,5 @@ jobs:
 
     - name: Test all models with onnx.checker, onnx.shape_inference, onnx.version_converter
       run: |
+        python -m pip install -q -r requirements-release_test.txt # necessary?
```

Comment:
I don’t think? What happens if you remove?

Reply:
Removed